### ISSUE TITLE
Error when restarting ERS container

### ISSUE THREAD

**Issue by p3k** [msokk/electron-render-service#31]
When I stop the ERS container with `docker stop electron-render-service` and then subsequently start it again with `docker run electron-render-service` I get the following error message in the logs:

```
2017-03-02T15:38:40.151926962Z xvfb-run: error: Xvfb failed to start
```

I currently circumvent this by simply removing the container and running a new one from the image – but would it be possible to make this work, anyway? Or is this a limitation of Docker?

**Comment by p3k** [comment]
Hi @msokk – no unfortunately, the error still persits. Here is what I did:

```
$ docker pull msokk/electron-render-service:latest
$ docker images msokk/*
REPOSITORY                      TAG                 IMAGE ID            CREATED             SIZE
msokk/electron-render-service   latest              3a3c3f8ae683        22 hours ago        626 MB
msokk/electron-render-service   <none>              a3838f69b08d        6 weeks ago         616 MB

$ docker run -d -e RENDERER_ACCESS_KEY=secret -p 3000:3000 msokk/electron-render-service
4ff9be83f060c03ab5bd8963a7a6b1040d1500c0c1ada4346b93ea93f2abdb2a

$ docker ps
CONTAINER ID        IMAGE                           COMMAND                  CREATED             STATUS              PORTS                    NAMES
4ff9be83f060        msokk/electron-render-service   "sh -c 'xvfb-run -..."   23 seconds ago      Up 22 seconds       0.0.0.0:3000->3000/tcp   stoic_sinoussi

$ docker stop stoic_sinoussi
stoic_sinoussi

$ docker start stoic_sinoussi 
stoic_sinoussi

$ docker logs -tf stoic_sinoussi 
2017-03-13T09:39:24.438680027Z Xlib:  extension "RANDR" missing on display ":99".
2017-03-13T09:39:25.290937800Z Xlib:  extension "RANDR" missing on display ":99".
2017-03-13T09:39:25.306642356Z [25:0313/093925.303051:ERROR:bus.cc(427)] Failed to connect to the bus: Failed to connect to socket /var/run/dbus/system_bus_socket: No such file or directory
2017-03-13T09:39:25.315184881Z [25:0313/093925.315039:WARNING:audio_manager.cc(317)] Multiple instances of AudioManager detected
2017-03-13T09:39:25.315202646Z [25:0313/093925.315053:WARNING:audio_manager.cc(278)] Multiple instances of AudioManager detected
2017-03-13T09:39:25.315752241Z libudev: udev_has_devtmpfs: name_to_handle_at on /dev: Operation not permitted
2017-03-13T09:39:25.374141879Z Renderer listening on http://172.17.0.2:3000
2017-03-13T09:39:25.374162715Z 
2017-03-13T09:39:25.375606872Z Usage: GET http://172.17.0.2:3000/[pdf|png|jpeg]?accessKey=[token]&url=http%3A%2F%2Fgoogle.com
2017-03-13T09:40:40.239363124Z xvfb-run: error: Xvfb failed to start
```

**Comment by Reggino** [comment]
Should be fixed by PR #39

**Comment by msokk** [comment]
Published the PR as 0.10.1
